src/changelog/unreleased.md

```diff
@@ -65,6 +65,7 @@ changelog entry.
   `ApplicationHandler::resumed/suspended()` are now only emitted by iOS and Web
   and now signify actually resuming/suspending the application.
 - Rename `platform::web::*ExtWebSys` to `*ExtWeb`.
+- `DeviceId::dummy()` and `WindowId::dummy()` are now not `unsafe`.
```

Comment:
Probably this is a bit better than double negation.

```suggestion
- Mark `DeviceId::dummy()` and `WindowId::dummy()` as safe.
```

Reply:
I intentionally avoided the word "safe" because its not 100% clear that we are talking about removing Rust `unsafe`.

```suggestion
- `DeviceId::dummy()` and `WindowId::dummy()` are no longer marked `unsafe`.
```
Doesn't remove the double negation, but I think using the verb "mark" makes it much easier to understand.